presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -118,6 +121,7 @@ private OrcWriterOptions(
         this.ignoreDictionaryRowGroupSizes = ignoreDictionaryRowGroupSizes;
         this.preserveDirectEncodingStripeCount = preserveDirectEncodingStripeCount;
         this.flattenedColumns = flattenedColumns;
+        this.dwrfStreamReorderingConfig = dwrfStreamReorderingConfig;
```

Comment:
We use streamLayoutFactory, passed into this config, to order streams. Did you have chance to figure out how existing streamLayoutFactory will work with the flat map ordering config?